### PR TITLE
[SOAR-6672] Add Timeout Preset

### PR DESCRIPTION
**Pull request by jrose-r7**
Jira link: https://issues.corp.rapid7.com/browse/SOAR-6672
### Description

Add a Timeout preset to the ConnectionTestException within the plugin runtime. 

Cause: "The connection timed out."

Assistance: "Verify network connectivity."

### Types of changes

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to change)

### Checklist:

- [ ] I have updated the documentation accordingly (if changes are required).
- [ ] I have tested the changes locally.

### CODE REVIEW

**Review by pmara-r7 — APPROVED** [review]
LGTM

**Comment by pmara-r7** [comment]
Seems like some of the CI stuff might be busted but the changes here look good

**Review by mrinehart-r7 — CHANGES_REQUESTED** [review]
Need to bump version & changelog and test locally

File: insightconnect_plugin_runtime/exceptions.py [hunk 1]
```diff
@@ -89,6 +92,8 @@ class Preset(object):
         Preset.UNKNOWN: "Check the logs and if the issue persists please contact support.",
         Preset.BASE64_ENCODE: "This is likely a programming error, if the issue persists please contact support.",
         Preset.BASE64_DECODE: "This is likely a programming error, if the issue persists please contact support.",
+        Preset.TIMEOUT: "This is likely a network error."
```
Comment (mrinehart-r7) [inline insightconnect_plugin_runtime/exceptions.py:95]:
```suggestion
        Preset.TIMEOUT: "This is likely a network error. "
```
```suggestion
        Preset.TIMEOUT: "This is likely a network error."
```

**Review by mrinehart-r7 — CHANGES_REQUESTED** [review]

File: insightconnect_plugin_runtime/exceptions.py [hunk 2]
```diff
@@ -58,6 +58,8 @@ class Preset(object):
         UNKNOWN = "unknown"
         BASE64_ENCODE = "base64_encode"
         BASE64_DECODE = "base64_decode"
+        TIMEOUT = "timeout"
+
```
Comment (mrinehart-r7) [inline insightconnect_plugin_runtime/exceptions.py:62]:
```suggestion
```

Comment (mrinehart-r7) [reply insightconnect_plugin_runtime/exceptions.py:62]: For PEP8 compliance

**Review by mrinehart-r7 — CHANGES_REQUESTED** [review]

**Comment by mrinehart-r7** [comment]
Screen Shot 2021-09-20 at 10 12 55 AM

**Review by pmara-r7 — APPROVED** [review]

File: insightconnect_plugin_runtime/exceptions.py [hunk 3]
```diff
@@ -89,6 +90,8 @@ class Preset(object):
         Preset.UNKNOWN: "Check the logs and if the issue persists please contact support.",
         Preset.BASE64_ENCODE: "This is likely a programming error, if the issue persists please contact support.",
         Preset.BASE64_DECODE: "This is likely a programming error, if the issue persists please contact support.",
+        Preset.TIMEOUT: "This is likely a network error."
```
Comment (mrinehart-r7) [inline insightconnect_plugin_runtime/exceptions.py:93]:
```suggestion
        Preset.TIMEOUT: "This is likely a network error. "
```

**Review by mrinehart-r7 — CHANGES_REQUESTED** [review]

**Review by mrinehart-r7 — APPROVED** [review]

**Review by mrinehart-r7 — APPROVED** [review]

**Pull request merged**